coderabbit.yaml

```
@@ -0,0 +1,163 @@
+  learnings:
+    scope: auto
+  issues:
+    scope: auto
+  jira:
+    usage: auto
+    project_keys: []
+  linear:
+    usage: auto
+    team_keys: []
+  pull_requests:
+    scope: auto
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)